gui/packages/ubuntupro/lib/pages/widgets/page_widgets.dart

```diff
@@ -168,3 +169,91 @@ class _PageContent extends StatelessWidget {
     );
   }
 }
+
+/// Two-column, vertically centered page. The left column always contains the
+/// svg image and title, with the left children below it. Both columns are equal
+/// in width. Optionally, a [NavigationRow] may be provided that will span the
+/// width below both columns.
+class ColumnPage extends StatelessWidget {
+  const ColumnPage({
+    required this.left,
+    required this.right,
+    this.svgAsset = 'assets/Ubuntu-tag.svg',
+    this.title = 'Ubuntu Pro',
+    this.navigationRow,
+    super.key,
+  });
+
+  final List<Widget> left;
+  final List<Widget> right;
+  final String svgAsset;
+  final String title;
+  final NavigationRow? navigationRow;
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+
+    return Pro4WSLPage(
+      body: Padding(
+        padding: const EdgeInsets.fromLTRB(32.0, 32.0, 32.0, 32.0),
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            Expanded(
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                children: [
+                  // Left column
+                  Expanded(
+                    child: Column(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: [
+                        RichText(
+                          text: TextSpan(
+                            children: [
+                              WidgetSpan(
+                                child: SvgPicture.asset(
+                                  svgAsset,
+                                  height: 70,
+                                ),
+                              ),
+                              const WidgetSpan(
+                                child: SizedBox(
+                                  width: 8,
+                                ),
+                              ),
+                              TextSpan(
+                                text: title,
+                                style: theme.textTheme.displaySmall
+                                    ?.copyWith(fontWeight: FontWeight.w100),
+                              ),
+                            ],
+                          ),
+                        ),
```

Comment:
Our existing format supports arbitrary titles/images, so we'd have to drop support for that and just use full logos with titles in them, instead of text. If you think that's better I can do that.

I'm not sure, but is there an accessibility concern with making text into images, like for logos?

Reply:
Good point about accessibility. Let's keep it as is then.